experiments/dorop-20/metadata/models.py

```diff
@@ -0,0 +1,81 @@
+from pathlib import Path
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+
+from pydantic import BaseModel
+
+class RecordStatus(Enum):
+    STAGE = "stage"
+    STORE = "store"
+
+@dataclass
+class Actor():
+    address: str
+    role: str
+
+@dataclass
+class PreservationEvent():
+    identifier: str
+    type: str
+    datetime: datetime
+    detail: str
+    actor: Actor
+
+class CommonMetadata(BaseModel):
+    title: str
```

Comment:
I left out `@schema` because I was having trouble getting it to play nice with Python and Pydantic.